06MPI/sec02PointToPoint.md

```diff
@@ -143,22 +147,32 @@ Why would we use Send instead of SSend?
 
 {% fragment send, cpp/point2point.cc %}
 
+---
+
 Both guarantee the buffer is safe to reuse. Send makes no guarantee as to
 whether it returns early or not. But *SSend* forces a *synchronisation point*:
 the codes reach the matching places, with all processes waiting until all reach
 that point.
 
 It may come out slightly faster to use Send,
-since having a **synchronisation point** when you don't need one can slow things
-down: Suppose (A) runs slightly faster, then (B) does; at the end, they've both
+since having a *synchronisation point* when you don't need one can slow things
```

Comment:
Technically it is bad practice to use italics in public presentations. Compared to using bold for emphasis it causes additional problems for a large class of dyslexic readers.

Reply:
I didn't know that. Will fix!

Reply:
Done (in lecture 6 at least). I'll do others as I see them.